### PR TITLE
[homework #9] - js-advanced-2 by Emiliano Acerbi

### PR DESCRIPTION
**Pull request by emiacerbi**
Hello @VitalikMelnyk . This is my first iteration of the 9th lecture. I'm having trouble in line 73 when I try to format the date, but all the other tests are passing.

Please let me know if there's anything to fix.

### CODE REVIEW

File: homeworks/emiliano-acerbi/09-js-advanced-2/index.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+const serialized = tolik.serialize();
+tolik = null;
+
+const resurrectedTolik = new UserDTO().wakeFrom(serialized);
+
+console.log(resurrectedTolik instanceof UserDTO); // true
+
+console.log(resurrectedTolik);
+
+// console.log(resurrectedTolik.printInfo()); // A. Nashovich - 2020327, 1999-01-02T00:00:00.000Z
```
Comment (solvd-mmelnyk) [inline homeworks/emiliano-acerbi/09-js-advanced-2/index.js:73]: looks like you have next error here:
`this.birth.toISOString is not a function`
I think it related that formatData method does not have handle for Date type

**Review by solvd-mmelnyk — COMMENTED** [review]
@emiacerbi good work!
in general looks good! 
Please fix problem with Date type and we will ready to go!

File: homeworks/emiliano-acerbi/09-js-advanced-2/index.js (hunk 1 above)
Comment (emiacerbi) [reply homeworks/emiliano-acerbi/09-js-advanced-2/index.js]: Hi Myk. I tried just loggin the date and it returns what the test required. Is it correct? Let me know if I should fix anything.

**Review by VitalikMelnyk — APPROVED** [review]

**Comment by VitalikMelnyk** [comment]
Well Done @emiacerbi.

**Pull request merged**